core/lib/config/src/configs/gateway.rs

```diff
@@ -30,13 +30,15 @@ pub struct GatewayChainConfig {
     pub diamond_proxy_addr: Address,
     pub chain_admin_addr: Option<Address>,
     pub governance_addr: Address,
+    pub current_settlement_layer: u64,
```

Comment:
right now this value is not the same as the value that is stored in the bridgehub, maybe it makes sense to make it represent the actual chain id of the settlement layer (i.e. it is 0 when it is L1)